### PR TITLE
Add selectedDateColor prop to DatePicker component

### PR DESCRIPTION
**Pull request by cerinman**
We have a need to be able to set the color used for the selected date and hover state of days on the calendar.  This PR adds that ability via a new prop.  

Also update the DatePicker component to use the Style constants file for colors/font sizes and stripped the px unit where possible.

New Prop

selectedDateColor Type(string) - Default('#359BCF') - Used to change the color of the selected date and hover state of days on the calendar.

@mxenabled/frontend-engineers 

screen shot 2015-12-09 at 4 20 11 pm

### CODE REVIEW

**Comment by cerinman** [comment]
Again, eslint is passing locally but failing here.

**Comment by bsbeeks** [comment]
you can take out the datepicker fullscreen component. I'm in the middle of a complete rewrite of it.

File: src/components/DatePicker.js [hunk 1]
```diff
@@ -132,7 +132,11 @@ class DatePicker extends React.Component {
         >
           <div
             key={startDate.format('DDDD')}
-            style={[styles.calendarDayContent, noSelectDay ? styles.calendarDayDisabled : isCurrentDay && styles.currentDay]}
+            style={[
+              styles.calendarDayContent,
+              noSelectDay ? styles.calendarDayDisabled : isCurrentDay && { backgroundColor: this.props.selectedDateColor, color: StyleConstants.Colors.INVERSE_PRIMARY },
+              { ':hover': { backgroundColor: this.props.selectedDateColor, color: StyleConstants.Colors.INVERSE_PRIMARY, cursor: 'pointer' } }
```
Comment (bsbeeks) [inline src/components/DatePicker.js]: could we define this in a variable above so we can format it with line breaks?

**Comment by cerinman** [comment]
@bsbeeks ready for another review

**Comment by bsbeeks** [comment]
:shipit:

**Pull request merged**